kuberwatcher.py

```diff
@@ -245,7 +245,7 @@ def main(es, defaults):
     namespaces = get_namespaces(defaults)
     pods = get_all_pods(namespaces)
     watches = generate_watch(pods)
-    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('reply_to', None))
+    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('throttle', 3600000), defaults.get('reply_to', None))
     watches['metricbeat']['metadata']['message'] = 'No metricbeat data has been recieved in the last 5 minutes! <{0}|kibana>'.format(defaults['kibana_url'])
```

Comment:
`last 5 minutes!` is no longer accurate? 

https://humanfriendly.readthedocs.io/en/latest/api.html#humanfriendly.format_timespan Could be quite cool, if pulling in another lib isn't an issue :)

Reply:
Ah, ignore that comment... Confusing `throttle` with `window`... :doh:

Reply:
Actually, I think there's still something there... 

Does overriding the `watcher.window` item actually change this `message`?

Reply:
> Does overriding the watcher.window item actually change this message?

 Nope, but I'm working on another PR to make sure that the window itself is actually configurable for the "no metricbeat data" alert and that the text is also updated properly ( including other useful optional metadata like cluster name).

Reply:
Cool, ok :)